src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs

```diff
@@ -563,11 +553,11 @@ private static bool PrepareHttpWebResponseReflectionObjects(Assembly systemNetHt
 
     private static void PerformInjection()
     {
-        FieldInfo servicePointTableField = typeof(ServicePointManager).GetField("s_ServicePointTable", BindingFlags.Static | BindingFlags.NonPublic)
-            ?? throw new InvalidOperationException("Unable to access the ServicePointTable field");
+        var servicePointTableField = typeof(ServicePointManager).GetField("s_ServicePointTable", BindingFlags.Static | BindingFlags.NonPublic)
+                                     ?? throw new InvalidOperationException("Unable to access the ServicePointTable field");
```

Comment:
Did dotnet format add this blank space? Interesting 🤔

Reply:
For sure, it is not complaining :)

It seems that it is alliggned to typeof from the previous line.